.github/release-drafter-draft-release-config.yml

```diff
@@ -0,0 +1,34 @@
+# Config file for release drafter
+name-template: 'v$RESOLVED_VERSION'
+tag-template: 'v$RESOLVED_VERSION'
+categories:
+  - title: 'Features'
+    labels:
+      - 'feature'
+      - 'enhancement'
+  - title: 'Bug Fixes'
+    labels:
+      - 'fix'
+      - 'bugfix'
+      - 'bug'
+  - title: 'Maintenance'
+    labels: 
+      - 'chore'
```

Comment:
I feel like "chore" is too generic, can we use labels like "maintenance", "documentation", "buildscripts" here?